pkg/pool-cow/contracts/CowRouter.sol

```diff
@@ -60,6 +72,16 @@ contract CowRouter is SingletonAuthentication, VaultGuard, ICowRouter {
         emit ProtocolFeePercentageChanged(newProtocolFeePercentage);
     }
 
+    /// @inheritdoc ICowRouter
+    function setFeeSweeper(address newFeeSweeper) external authenticate {
+        if (newFeeSweeper == address(0) || newFeeSweeper == address(this)) {
```

Comment:
As noted above, do we need to check for our own address here, or just zero? (In which case we wouldn't need the address argument to the error.)